gofile-downloader.py

```diff
@@ -377,6 +379,17 @@ def _download(self, url: str, password: str | None = None) -> None:
             self._resetClassProperties()
             return
 
+        interactive: bool | None = getenv("GF_INTERACTIVE") == "1"
+        if interactive:
+            self._listFiles()
+            input_list: list[str] = input("Enter the index of the files to download separated with spaces: ").split()
+            input_list = list(set(input_list) & set(self._files_info.keys())) # ensure only valid index strings are stored
+
+            if input_list:
```

Comment:
Just some feedback to the user in case invalid keys were used.

```patch
-            if input_list:
-                keys_to_delete: list[str] = list(set(self._files_info.keys()) - set(input_list))
-                for key in keys_to_delete:
-                    del self._files_info[key]
-                    
+            if not input_list:
+                _print(f"Nothing done.{NEW_LINE}")
+                rmdir(self._content_dir)
+                self._resetClassProperties()
+
+                return
+
+            keys_to_delete: list[str] = list(set(self._files_info.keys()) - set(input_list))
+
+            for key in keys_to_delete:
+                del self._files_info[key]
```